frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -1,464 +1,204 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
 import * as PropTypes from 'prop-types';
-import {CatalogTileView} from 'patternfly-react-extensions/dist/js/components/CatalogTileView';
 import {CatalogTile} from 'patternfly-react-extensions/dist/js/components/CatalogTile';
-import {VerticalTabs} from 'patternfly-react-extensions/dist/js/components/VerticalTabs';
-import {FilterSidePanel} from 'patternfly-react-extensions/dist/js/components/FilterSidePanel';
-import {EmptyState} from 'patternfly-react/dist/js/components/EmptyState';
-import FormControl from 'patternfly-react/dist/js/components/Form/FormControl';
 import {Modal} from 'patternfly-react/dist/js/components/Modal';
 
+import {history} from '../utils/router';
 import {normalizeIconClass} from './catalog-item-icon';
-import {categorizeItems, recategorizeItems} from '../utils/categorize-catalog-items';
 import {CatalogTileDetails} from './catalog-item-details';
-import {history} from '../utils';
+import {TileViewPage} from '../utils/tile-view-page';
+
+export const catalogCategories = {
```

Comment:
> Categories are dynamically created.

Doesn't look like it?

Reply:
Yeah, bad comment. Will fix, thanks.

Reply:
That comment is for `Marketplace` categories which is the addition in the PR (catalog was refactored to use common code with marketplace).

Reply:
Updated the comment to be more specific